code/controllers/subsystem/vote.dm

```diff
@@ -177,8 +177,11 @@ SUBSYSTEM_DEF(vote)
 		switch(vote_type)
 			if("restart")
 				choices.Add("Restart Round", "Continue Playing")
-			if("gamemode")
-				choices.Add(config.votable_modes)
+        	if("gamemode")
+            	if(length(GLOB.clients) > 35)
+                	to_chat(usr, "<span class='warning'>The player count is over 35, so public gamemode voting has been disabled.</span>")
+                	return FALSE
+                choices.Add(config.votable_modes)
```

Comment:
Does not compile
Make the vote pop cap a config option
Let admins override the cap

Reply:
![image](https://user-images.githubusercontent.com/44979174/95447097-260c1f00-0959-11eb-8247-9c49be9797e5.png)

![image](https://user-images.githubusercontent.com/44979174/95447112-2ad0d300-0959-11eb-9b5b-9f0b29c6e756.png)

![image](https://user-images.githubusercontent.com/44979174/95447212-52c03680-0959-11eb-84ba-f8b49abf61d8.png)

That is way beyond my means, this is only a stopgap because nobody else will